pkg/controller/migmigration/backup.go

```diff
@@ -88,8 +142,20 @@ func (t Task) getBackup() (*velero.Backup, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Find proper backup to return
 	if len(list.Items) > 0 {
```

Comment:
Why test the length of the list.Items before iterating?  The _range_ on an empty collection will have the same result.